src/ampJson.js

```diff
@@ -1,7 +1,14 @@
-export default function ampJson(data, component) {
+export default function ampJson(data, component, attributes={}) {
   if (!data || !component) return '';
+
+  const attributesString = attributes ?
+    Object.entries(attributes).map(([key, val]) => (
```

Comment:
this does not escape either `key` or `val`, either of which could contain quotes.